### PR TITLE
Add tsplib95 and python-tsp.

### PR DESCRIPTION
**Pull request by moorepants**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by moorepants** [comment]
ping @conda-forge/help-python this one should need minimal review, just two noarch packages with nothing complex.

File: recipes/python-tsp/a280.tsp [hunk 1]
```diff
@@ -0,0 +1,287 @@
+NAME : a280
```
Comment (isuruf) [inline recipes/python-tsp/a280.tsp:1]: Can we use a file from the sdist? Something like `tests/data/att532.tsp`?

Comment (moorepants) [reply recipes/python-tsp/a280.tsp:1]: Yes, I can look into it.

Comment (moorepants) [reply recipes/python-tsp/a280.tsp:1]: There are no data files in the sdist:

```
moorepants@nandi:Downloads$ cd python_tsp-0.4.0/
moorepants@nandi:python_tsp-0.4.0$ tree
.
├── LICENSE
├── PKG-INFO
├── pyproject.toml
├── python_tsp
│   ├── distances
│   │   ├── data_processing.py
│   │   ├── euclidean_distance.py
│   │   ├── great_circle_distance.py
│   │   ├── __init__.py
│   │   ├── osrm_distance.py
│   │   └── tsplib_distance.py
│   ├── exact
│   │   ├── branch_and_bound
│   │   │   ├── __init__.py
│   │   │   ├── node.py
│   │   │   ├── priority_queue.py
│   │   │   └── solver.py
│   │   ├── brute_force.py
│   │   ├── dynamic_programming.py
│   │   └── __init__.py
│   ├── heuristics
│   │   ├── __init__.py
│   │   ├── local_search.py
│   │   ├── perturbation_schemes.py
│   │   └── simulated_annealing.py
│   ├── __init__.py
│   └── utils
│       ├── __init__.py
│       ├── permutation_distance.py
│       └── setup_initial_solution.py
└── README_pypi.rst
```

**Comment by moorepants** [comment]
ping @conda-forge/help-python can this be merged? I can ask upstream to include the data files for a future release.

**Comment by xhochy** [comment]
If the file is available via git(hub) you could either use the tarball or for a single file from github the raw.githubcontent.com URL.

**Comment by moorepants** [comment]
What is the issue about including this file? I've included files like this in many other feedstocks over the years.

**Comment by moorepants** [comment]
Hi, I removed this data file for the package test and download it instead from the raw git url. Is there anything else I need to adjust? Thanks.

**Comment by moorepants** [comment]
@conda-forge/help-python I've tried to address all comments. I think this PR is ready, can it be merged? Thanks.

**Comment by moorepants** [comment]
Dear @conda-forge/help-python, can this PR be merged? It is a simple noarch copy of a PyPi package.

**Comment by moorepants** [comment]
Thanks @ocefpaf !

**Pull request merged**